header/p2p/exchange.go

```diff
@@ -75,13 +75,13 @@ func (ex *Exchange) Start(context.Context) error {
 	ex.ctx, ex.cancel = context.WithCancel(context.Background())
 
 	go ex.peerTracker.gc()
-	go ex.peerTracker.track()
 	for _, p := range ex.trustedPeers {
 		// Try to pre-connect to trusted peers.
 		// We don't really care if we succeed at this point
 		// and just need any peers in the peerTracker asap
 		go ex.host.Connect(ex.ctx, peer.AddrInfo{ID: p}) //nolint:errcheck
 	}
+	go ex.peerTracker.track()
```

Comment:
Does the order matters?

Reply:
It's any way up to the scheduler to decide what goroutine will be picked first. 

Just grouping wise, pls keep track and gc `go`s together.